jot/JotTextEditView.m

```diff
@@ -123,6 +123,45 @@ - (instancetype)init
             make.width.equalTo(@32);
             make.height.equalTo(@20);
         }];
+
+        NSBundle *bundle = [NSBundle bundleForClass:[self class]];
+        NSURL *url = [bundle URLForResource:@"jot" withExtension:@"bundle"];
+        
+        self.trashcanButton = [[UIButton alloc] init];
+        [self.trashcanButton setImage:[UIImage imageNamed:@"trashcan" inBundle:[NSBundle bundleWithURL:url] compatibleWithTraitCollection:nil] forState:UIControlStateNormal];
```

Comment:
woah, interesting way to import an image. Does it not just work with the Asset catalog?

Reply:
Yeah Cocapods recommends defining a resource bundle that's defined in the `podspec`: http://guides.cocoapods.org/syntax/podspec.html#resources